example/package.json

```diff
@@ -13,10 +13,9 @@
     "clean": "gatsby clean"
   },
   "dependencies": {
-    "@piwikpro/gatsby-plugin-piwik-pro": "file:../",
+    "@piwikpro/gatsby-plugin-piwik-pro": "file:../plugin",
```

Comment:
how to resolve this from package downloaded from npm?

Reply:
I'm not sure what you mean. If you want to use this package from npm, you should use `@piwikpro/gatsby-plugin-piwik-pro": "1.0.0"`.